clients/linux/src/menubar.rs

```diff
@@ -174,17 +190,17 @@ impl Item {
     }
 
     #[rustfmt::skip]
-    fn data() -> Vec<(Self, ItemData)> {
+    fn data() -> Vec<(Self, Option<String>, ItemData)> {
```

Comment:
Why not do a static string (`&'static str`) type here instead of a `Option<String>`? Seems a bit cluttered